### DISCUSSION TITLE
Brand New UI

### DISCUSSION THREAD

**Discussion by Cotopia-Inc** [JellyChain1/JellyChain1#11] [Announcements]
The New website is now live. It is a demo of the software we build. Let us know what you think.

**Comment by GoGoBand** [comment]
love what it look like

**Reply by Cotopia-Inc** [reply]
thanks

**Comment by billyjo111** [comment]
Love the sliders on the UI

**Reply by Cotopia-Inc** [reply]
thanks

**Comment by Cotopia-Inc** [comment]
This new UI will serve as the Official Pitch Deck for JellyChain1.

**Reply by billyjo111** [reply]
I love it. the UI is easy to read and People can get to the get hub very easy from the UI.
